### DISCUSSION TITLE
Is there a way to add one pictures to each leaf page, displayed on the branch page?

### DISCUSSION THREAD

**Discussion by kevinlinxc** [jpanther/congo#336] [General]
So leave pages can easily have images inside them, but it seems like the branch page only has text preview?

This is what I'm imagining, just a standard thumbnail to go with each leaf pages title:
![image](https://user-images.githubusercontent.com/44324811/197934057-17206b45-710e-48cc-b72e-1145c53234f6.png)

**Comment by kevinlinxc** [comment]
Same as: https://github.com/jpanther/congo/discussions/182 and https://github.com/jpanther/congo/discussions/315, both of which are unanswered.

**Reply by jpanther** [reply]
Thanks for the suggestion. This is something that I have considered adding to the theme but have not had time to implement as yet. I may look to add this to a future release.

**Reply by jpanther** [reply]
Congo v2.4.0 now includes article thumbnail support. The [getting started docs](https://jpanther.github.io/congo/docs/getting-started/#feature-cover-and-thumbnail-images) have more details on how this new feature works.
